driver-core/src/test/resources/command-monitoring/bulkWrite.json

```diff
@@ -110,6 +110,11 @@
       "description": "A successful unordered bulk write with an unacknowledged write concern",
       "operation": {
         "name": "bulkWrite",
+        "collectionOptions": {
```

Comment:
Will see if we need a SPEC ticket for this

Reply:
I imagine it will as its changed the test, so people will need to update their runners to use it.